frontend/lib/src/theme/globalStyles.ts

```diff
@@ -259,56 +248,6 @@ export const globalStyles = (theme: EmotionTheme): SerializedStyles => css`
     font-weight: ${theme.fontWeights.extrabold};
   }
 
-  // Mark
-
-  mark {
-    padding: 0.2em;
-    background-color: ${theme.colors.secondaryBg};
-  }
-
-  // Sub and Sup
-  //
-  // Prevent sub and sup elements from affecting the line height in
-  // all browsers.
-
-  sub,
-  sup {
-    position: relative;
-    line-height: 0;
-    vertical-align: baseline;
-  }
-
-  sub {
-    bottom: -0.25em;
-  }
-  sup {
-    top: -0.5em;
-  }
-
-  // Links
-
-  a {
-    color: ${theme.colors.primary};
-    text-decoration: underline;
-
-    &:hover {
-      color: ${darken(theme.colors.primary, 0.15)};
-    }
-  }
-
-  // And undo these styles for placeholder links/named anchors (without href).
-  // It would be more straightforward to just use a[href] in previous block, but that
-  // causes specificity issues in many other styles that are too complex to fix.
-  // See https://github.com/twbs/bootstrap/issues/19402
-
-  a:not([href]):not([class]) {
-    &,
-    &:hover {
-      color: inherit;
-      text-decoration: none;
-    }
-  }
```

Comment:
This does not seem to be ported over to styled-components. Was this never used?

Reply:
I reverted this for now. I will look into this in a subsequent PR